code_pylint.py

```diff
@@ -54,7 +54,7 @@
                                 'too-many-return-statements', 'import-outside-toplevel',
                                 'too-many-instance-attributes', 'broad-except', 'bare-except', "broad-exception-caught",
                                 'too-many-public-methods', 'too-many-statements', 'undefined-loop-variable',
-                                'attribute-defined-outside-init']
+                                'attribute-defined-outside-init', 'wrong-spelling-in-docstring']
```

Comment:
Wrong spelling in docstring should not be added to ERRORS_WITHOUT_TIME_DECREASE as the aim of it is to lower the number of spelling errors over time...

Reply:
ok,  I changed that